patches/abort-controller+3.0.0.patch

```diff
@@ -13,6 +13,7 @@ index b0c5ec3..c8c8018 100644
 +const { AbortController } = globalThis;
  /*eslint-enable @mysticatea/prettier */
  
- module.exports = AbortController
+-module.exports = AbortController
 -module.exports.AbortSignal = AbortSignal
 -module.exports.default = AbortController
++module.exports = { AbortController }
```

Comment:
This is required because this [PR](https://github.com/MetaMask/controllers/pull/715) (part of controllers v30.0.0) changed an import of `AbortController` [from a default import to a named import](https://github.com/MetaMask/controllers/pull/808/files#diff-c2ec75b01a3b2e2e1284df7c48ad4c80a26ea7f1455167cc8390d91ca865b854R3), which breaks on the extension without modifying this patch.